README.md

```diff
@@ -10,6 +10,12 @@ There are dependencies needed for running Deploy and marketplace imposes restric
 
 ## Building
 
+Dependencies
+
+ * [Operator SDK](https://sdk.operatorframework.io/)
+   * Tested with 1.8.0
```

Comment:
Worth noting there is also a line at the very beginning that says `Built with operator-sdk (v1.7.2)`

Reply:
will fix